Assets/Scripts/Joystick/InputService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InputService : MonoSingleton<InputService>
+{
+    private Joystick joystick;
+    public Joystick Joystick{
+        get{
+            return joystick;
+        }
+    }
+
+    public Joystick tempJoystick;
```

Comment:
Should not be public. Change to private serialized field if you want to populate from inspector